### PR TITLE
feat: implementation of the simple integral computation function

### PR DESCRIPTION
**Pull request by Vivano**

### CODE REVIEW

File: calculus/api.ts [hunk 1]
```diff
@@ -48,4 +48,35 @@ export function evaluate_function(fx: string, variables : object) : APIResult {
             error: e.message
         }
     }
-}
\ No newline at end of file
+}
+
+
+
+//
```
Comment (QuentinRa) [inline calculus/api.ts:55]: Remove this

File: calculus/api.ts [hunk 2]
```diff
@@ -48,4 +48,35 @@ export function evaluate_function(fx: string, variables : object) : APIResult {
...
+// @ts-ignore
+import {create,all} from "mathjs";
+const math = create(all);
+
+import * as integral from "./lib/integral";
+
+// @ts-ignore
+math.import( [[integral.createIntegral]] ); //simulates importing the npm package, which nests it [[]]
+
+export function compute_integral(fx: string, x: string) {
```
Comment (QuentinRa) [inline calculus/api.ts:69]: Comment please

File: calculus/tests.js [hunk 3]
```diff
@@ -79,4 +79,38 @@ describe('evaluate_function', function() {
...
+       const x = 'x';
+       let res = compute_integral(fx,x);
+       assert.equal(res.result,"10 * y * x + x ^ 3 * 4 / 3");
+       assert.equal(res.error, null);
+   });
+   it('test with missing variable', function() {
+       const fx = '10y+4x^2';
+       let res = compute_integral(fx);
+       assert.equal(res.result,null);
+       assert.equal(res.error, "Unexpected type of argument in function integral (expected: string or SymbolNode or number or boolean, actual: undefined, index: 1)");
```
Comment (QuentinRa) [inline calculus/tests.js:100]: Check the start, not the whole message

File: calculus/tests.js [hunk 4]
```diff
@@ -79,4 +79,38 @@ describe('evaluate_function', function() {
...
+      let res = compute_integral(fx,z);
+      assert.equal(res.result,"(10 y + 4 x ^ 2) * z");
+      assert.equal(res.error, null);
+   });
+   it('test with null function', function() {
+       const fx = null;
+       const x = 'x';
+       let res = compute_integral(fx,x);
+       assert.equal(res.result, null);
+       assert.equal(res.error, "Unexpected type of argument in function integral (expected: string or Node or number or boolean, actual: null, index: 0)");
```
Comment (QuentinRa) [inline calculus/tests.js:114]: Same, check the start, not the whole message

File: calculus/tests.js (hunk 3 above)
Comment (Vivano) [reply calculus/tests.js:100]: ok done

File: calculus/tests.js (hunk 4 above)
Comment (Vivano) [reply calculus/tests.js:114]: same, done ;)

File: calculus/api.ts (hunk 1 above)
Comment (Vivano) [reply calculus/api.ts:55]: done

File: calculus/api.ts (hunk 2 above)
Comment (Vivano) [reply calculus/api.ts:69]: done

**Review by DataSaiyentist — APPROVED** [review]
Optimized according to Quentin's remarks.
Tested and approved

**Review by QuentinRa — APPROVED** [review]

**Pull request merged**